api/jobs.yaml

```diff
@@ -33,60 +33,56 @@ paths:
       tags:
         - Capabilities
 
-  '/jobs/{id}/abort':
-    post:
-      operationId: AbortJob
-      summary: Abort a job by ID
+  '/jobs/operationDetails':
```

Comment:
Nit, but what do you think about adding a comment in the Readme, or at the beginning of this file to let future dev know about how the endpoints need to be ordered?